imap/imapd.c

```diff
@@ -15056,6 +15056,11 @@ static void cmd_compress(char *tag, char *alg)
         prot_printf(imapd_out,
                     "%s OK %s active\r\n", tag, alg);
 
+        syslog(LOG_INFO, "compression enabled sessionid=<%s> userid=<%s> algorithm=<%s>",
+                         session_id(),
+                         imapd_userid ? imapd_userid : "",
+                         alg);
+
```

Comment:
If you're gonna use the foo=<bar> logging style, you might as well just use `xsyslog()` :)

Reply:
Okay, done!